apps/krew-website/src/scripts/script.js

```diff
@@ -0,0 +1,96 @@
+const navbar = document.querySelector('#navbar');
+
+// Darkmode switch
+if (localStorage.theme === 'dark' || (!'theme' in localStorage && window.matchMedia('(prefers-color-scheme: dark)').matches)) {
+  document.querySelector('html').classList.add('dark');
+} else if (localStorage.theme === 'dark') {
+  document.querySelector('html').classList.add('dark');
+}
+
+var lightSwitches = document.getElementsByClassName('js-light-switch');
+
+for (let i = 0; i < lightSwitches.length; i++) {
+  lightSwitches[i].addEventListener('click', function() {
+    //console.log("Clicked index: " + i);
+    let htmlClasses = document.querySelector('html').classList;
+    if (localStorage.theme == 'dark') {
+      htmlClasses.remove('dark');
+      localStorage.removeItem('theme');
+    } else {
+      htmlClasses.add('dark');
+      localStorage.theme = 'dark';
+    }
+  });
+}
+
+// document.getElementsByClassName('js-light-switch').addEventListener('click', function() {
+//   let htmlClasses = document.querySelector('html').classList;
+//   if (localStorage.theme == 'dark') {
+//     htmlClasses.remove('dark');
+//     localStorage.removeItem('theme');
+//   } else {
+//     htmlClasses.add('dark');
+//     localStorage.theme = 'dark';
+//   }
+// });
+
+// scroll to top with easing
+const back2Top = document.querySelector('#back2Top');
+
+const easeOutCubic = function(t) {
+  return (--t) * t * t + 1;
+};
+
+const scrollToTop = (startTime, currentTime, duration, windowScrollY) => {
+  const runtime = currentTime - startTime;
+  let progress = runtime / duration;
+  progress = Math.min(progress, 1);
+  const ease = easeOutCubic(progress);
+
+  window.scroll(0, windowScrollY + windowScrollY * -1 * ease);
+  if (runtime < duration) {
+    requestAnimationFrame((timestamp) => {
+      const currentTime = timestamp || new Date().getTime();
+      scrollToTop(startTime, currentTime, duration, windowScrollY);
+    });
+  }
+};
+
+back2Top.addEventListener('click', (e) => {
+  e.preventDefault();
+  const htmlEl = document.querySelector('html');
+  htmlEl.style.scrollBehavior = 'smooth';
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll

You can just pass `behavior` to the scroll API.